src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vsixmanifest

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="16.99.99" Language="en-US" Publisher="Microsoft Corporation" />
-        <DisplayName>NuGet Package Manager for Visual Studio 2019</DisplayName>
+        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="17.99.99" Language="en-US" Publisher="Microsoft Corporation" />
+        <DisplayName>NuGet Package Manager for Visual Studio 2022</DisplayName>
         <Description>A collection of tools to automate the process of downloading, installing, upgrading, configuring, and removing packages from a VS Project.</Description>
         <PackageId>Microsoft.VisualStudio.NuGet.Core</PackageId>
         <License>eula.rtf</License>
         <AllowClientRole>true</AllowClientRole>
     </Metadata>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[17.0,18.0)" DisplayName="Visual Studio core editor" />
     </Prerequisites>
     <Installation SystemComponent="true" InstalledByMsi="false" AllUsers="true">
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[16.0,17.0)" />
```

Comment:
I am curious to learn, why did we remove all the installation targets except for `Community` identifier.

Reply:
I think you're right that this was a mistake. I based this off of https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/21382/How-to-upgrade-VSIX-to-Dev17, and assumed that their example was exactly how they were suggesting we configure the manifest for Dev17, but looking at it again I think they just used a single InstallationTarget for simplicity.

Reply:
Once this PR is merged to `dev`, our manual test team will install the VSIX from the latest `dev` build to execute tests. With the current installation targets defined in the PR, AFAIK the test team can't install the VSIX on VS 2022 enterprise version. I think we should add the installation targets in this PR. Please let me know if my understanding is incorrect.

Reply:
CTI has already been doing manual testing of release-6.0.x builds installed into VS main. So it seems that the VSIX installer has no issue with installing a community-only VSIX into Enterprise.

In any case, I already added back the other SKUs (Pro, express, etc) for safety.